### ISSUE TITLE
Fix remove purchase order error

### ISSUE THREAD

**Issue by disanyixing** [Nuklon/Steam-Economy-Enhancer#185]
![image](https://github.com/Nuklon/Steam-Economy-Enhancer/assets/72839560/15edcdb7-6f92-44cb-b24a-6ab4ed2e8a51)

```
SteamMarket.prototype.removeListing = function (item, callback /*err, data*/) {
        var sessionId = readCookie('sessionid');
            $.ajax({
                type: "POST",
                url: window.location.protocol + '//steamcommunity.com/market/removelisting/' + item,
                data: {
                    sessionid: sessionId
                },
                success: function (data) {
                    callback(ERROR_SUCCESS, data);
                },
                error: function () {
                    // return callback(ERROR_FAILED);
                    $.ajax({
                        type: "POST",
                        url: window.location.protocol + '//steamcommunity.com/market/cancelbuyorder/',
                        data: {
                            sessionid: sessionId,
                            buy_orderid: item
                        },
                        success: function (data) {
                            callback(ERROR_SUCCESS, data);
                        },
                        error: function () {
                            return callback(ERROR_FAILED);
                        },
                        crossDomain: true,
                        xhrFields: {
                            withCredentials: true
                        },
                        dataType: 'json'
                    });
                },
                crossDomain: true,
                xhrFields: {
                    withCredentials: true
                },
                dataType: 'json'
            });
    };
```
Before fix：
![1%_)65AA1{(WLB8J483K)7A](https://github.com/Nuklon/Steam-Economy-Enhancer/assets/72839560/d47a658c-aa18-435f-850d-86cbf6886b66)

After fix：
![~III4R0F~K R14WZ`8N@JK0](https://github.com/Nuklon/Steam-Economy-Enhancer/assets/72839560/533a08df-7c11-48cd-ab5d-bdbcc79dc33d)

![NG X% L`9TCT46UDGQ)FR}O](https://github.com/Nuklon/Steam-Economy-Enhancer/assets/72839560/267f9d42-f48b-4f81-a117-a46fb1678553)

**Comment by xPaw** [comment]
Fixed by https://github.com/Nuklon/Steam-Economy-Enhancer/commit/2f0ae1e936fbf6d3be7ff90fb029c2b7b3a8fdef

**Comment by disanyixing** [comment]
> Fixed by [2f0ae1e](https://github.com/Nuklon/Steam-Economy-Enhancer/commit/2f0ae1e936fbf6d3be7ff90fb029c2b7b3a8fdef)

Nice fix. My fix is just to make the removing purchase order work successfully in exceptional circumstances. Your Fix makes the removing purchase order work directly.

**Comment by xPaw** [comment]
you can close this
